aten/src/TH/generic/THStorage.h

```diff
@@ -2,6 +2,8 @@
 #define TH_GENERIC_FILE "generic/THStorage.h"
 #else
 
+#include <ATen/Storage.h>
```

Comment:
This file is supposed to be C compatible, so this has to be protected with `ifdef __cplusplus`